### DISCUSSION TITLE
E-mail settings

### DISCUSSION THREAD

**Discussion by TommsNL** [liedekef/events-made-easy#513] [Q&A] [answered]
I'd like to send less e-mails when people are registering. Due limitations of our host we are forced to send in small batches what will result in long waiting times for receiving mail.

I'd like to disable for example the mail what is send to me (the organisator of the event). But i'd like to keep sending mail to all people that are registering.

Am i correct that this can be done by this setting?:
EME -> Settings -> E-Mail Tab -> Enable receiving RSVP email notifications?

_Were live with the event so I don't want to cause any downtime by wrong settings ;)_

**Comment by liedekef** [comment, marked as answer]
If you disable RSVP emails, the booker will also no longer receive mails.
Go into the EME settings => "Email templates" section, there for the mails you no longer want: set an empty subject (or body). It is also explained there: "If you leave this empty, this mail will not be sent." ...

**Reply by TommsNL** [reply]
I'm glad i double checked :D... Thanks!
